conda/recipes/libcudf/recipe.yaml

```diff
@@ -0,0 +1,297 @@
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ compiler("c") }}
+      host:
+        - cuda-version =${{ cuda_version }}
+        - libkvikio =${{ minor_version }}
+        - rapids-logger =0.1
+        - zlib ${{ zlib_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+          else: cuda-cudart-dev
+      run:
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - if: cuda_major == "11"
+          then:
+            - cudatoolkit
+            - if: linux64
+              then:
+                - libcufile ${{ cuda11_libcufile_run_version }}
+          else:
+            - cuda-nvrtc
+            - if: linux64
+              then:
+                - libcufile
+        - nvcomp ${{ nvcomp_version }}
+        - librmm =${{ minor_version }}
+        - libkvikio =${{ minor_version }}
+        - dlpack ${{ dlpack_version }}
+        - rapids-logger =0.1
+      run_exports:
+        - ${{ pin_subpackage("libcudf", upper_bound="x.x") }}
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-nvrtc
+          - cuda-nvtx
+          - cuda-version
+          - flatbuffers
+          - libcurand
+          - libkvikio
+          - librdkafka
+          - librmm
+          - nvcomp
+    tests:
+      - script:
+        - test -f $PREFIX/lib/libcudf.so
+        - test -f $PREFIX/include/cudf/column/column.hpp
+    about:
+      homepage: ${{ load_from_file("python/libcudf/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libcudf/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libcudf/pyproject.toml").project.description }}
+
+  - package:
+      name: libcudf_kafka
+      version: ${{ version }}
+    build:
+      script:
+        - cmake --install cpp/libcudf_kafka/build
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ stdlib("c") }}
+      host:
+        - librdkafka ${{ librdkafka_version }}
+        - ${{ pin_subpackage("libcudf", exact=True) }}
+      run:
+        - librdkafka ${{ librdkafka_version }}
+        - ${{ pin_subpackage("libcudf", exact=True) }}
+      ignore_run_exports:
+        by_name:
+          - cuda-cudart
+          - cuda-nvrtc
+          - cuda-nvtx
+          - cuda-version
+          - flatbuffers
+          - libcurand
+          - libkvikio
+          - librdkafka
+          - librmm
+          - nvcomp
+    tests:
+      - script:
+        - test -f $PREFIX/lib/libcudf_kafka.so
+    about:
+      homepage: https://rapids.ai/
+      license: Apache-2.0
+      summary: libcudf_kafka library
+
+  - package:
+      name: libcudf-example
+      version: ${{ version }}
+    build:
```

Comment:
How'd we get away without an `env:` section here? In https://github.com/rapidsai/rmm/pull/1800/ it seemed like I needed one...

Reply:
blind luck, I wager

Reply:
we got away with it because there were enough required dependencies hanging around in the CI environment, but now the proper dependencies (and build system) are set up via the `env` section in this output